src/platform/qemu-riscv64-virt/virt_desc.c

```diff
@@ -26,6 +26,8 @@ struct platform platform = {
         #else 
         #error "unknown IRQC type " IRQC
         #endif
+        // Disable ACLINT by default
+        .aclint_sswi.base = 0
```

Comment:
Could we add some kind of option compile time option to enable it similar to how we select the interrupt controller?